source/attributes/data-occlusion.md

```diff
@@ -0,0 +1,13 @@
+# data-occlusion
+
+```html
+<mr-app data-occlusion="..????"> ... </mr-app>
+```
+
+Occlusion setup can be attached to an [`mr-app`](/doc/mr-app) using `data-occlusion`:
+
+### `data-occlusion`
+
+... not sure what to put here but it seems to do with the anchor system?
```

Comment:
LOL. I was gonna say, I don't know what data-occlusion is, and was gonna ask for more details.

Reply:
XD def something i'm passing to @michaelthatsit atm from legacy setup